src/K9OCRS/ClientApp/src/areas/applications/Catalog/Classes.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import * as classTypesClient from '../../../util/apiClients/classTypes';
+import { Container, Spinner, Row, Col } from "reactstrap";
+import Table from "../../../shared/components/Table";
+import sectionColumns from './components/sectionColumns';
+import PageHeader from '../../../shared/components/PageHeader';
+import './components/style.scss';
+
+const Classes = props => {
+    const { classTypeId } = useParams();
+    const [classDetail, setClassDetail] = useState([]);
+    const [loading, setLoading] = useState(true);
+    const [alerts, setAlerts] = useState([]);
+
+    useEffect(() => {
+        async function getTest() {
+          if (classTypeId) {
+            try {
+              const res = await classTypesClient.getClassTypeByID(classTypeId);
+              setClassDetail(res?.data);
+              setLoading(false);
+            } catch(err) {
+              setLoading(false);
+              setAlerts([{ color: 'danger', message: 'We\'re having issues getting the details for this class' }]);
+            }
+          }   
+        }
+        getTest();
+      }, [classTypeId]);
+
+      const photoArr = classDetail.photos;
+
+
+    return (
+        <>
+        <PageHeader
+          title={classDetail.title ?? 'Class Details'}
+          alerts={alerts}
+          breadCrumbItems={[
+            { label: 'Class Catalog', path: '/' },
+            { label: classDetail.title ?? 'Class Details', active: true },
+          ]}
+        />
+        
+        <p>Class Catalog / {classDetail.title}</p>
+        <Container className="px-sm-5 container-sm" fluid>
```

Comment:
The p is replaced by the `PageHeader` breadcrumbs so it should be removed
```suggestion
        />
        <Container className="px-sm-5 container-sm" fluid>
```